docs/EMERGENCY.md

```diff
@@ -0,0 +1,213 @@
+# Emergency Procedures for Yearn Finance
+
+## TLDR: [Emergency Check List](#emergency-checklist)
+
+## Introduction
+
+This document details the emergency procedures and guidelines that should take place during an emergency situation.
+
+The main goal of this document is to help minimize as much as possible a potential loss of funds in yearn platform and Smart Contracts during an emergency situation.
+
+## Definitions and Examples of Emergencies
+
+For the purposes of this document we define emergency situation to be:
+
+**_Any situation that may lead to a considerable amount of loss of funds for yearn users._**
+
+This is a non exhaustive list of possible emergency scenarios:
+
+1. Exploit in Vault/Strategy code that can cause a loss of funds for users
+2. Bug in Vault/Strategy code that may cause loss of funds for users
+3. An exploit in an underlying protocol that a yearn strategy is utilizing that may lead to loss of funds for the users
+4. A bug in an underlying protocol that a yearn strategy is utilizing that may lead to loss of funds for the users
+5. A key role like a strategist having the private keys stolen
+
+## Roles
+
+The following roles are identified and should be assigned during an emergency.
+
+- Facilitator
+- Strategist Lead
+- Multi-sig Operations
+- Web Lead
+- Core Dev Lead (Guardian)
+- Ops
+
```

Comment:
Someone should take care of press/twitter communication if the issue has been disclosed.